### DISCUSSION TITLE
firmware update DOES work on macOS

### DISCUSSION THREAD

**Discussion by pleblancq** [kpine/zwave-js-server-docker#31] [General]
Hello, I never saw this [notice](https://github.com/kpine/zwave-js-server-docker/wiki/700-series-Controller-Firmware-Updates-(Linux)#known-working-platforms) about updates not working on macOS but it does work.

I updated twice, one in september 2022 when I received my Zooz ZST10 700 and one yesterday.

always have been macOS Big Sur (I don't remember version number in september) but last update:
- Big Sur 11.7.2 on x86
- minicom (brew) version 2.8 (compiled Jan  4 2021)
- lrzsz: stable 0.12.20 (bottled)

using /dev/tty.usbserial-[usb_device_id] instead of /dev//dev/cu.usbserial-[usb_device_id] as in the [issue](https://github.com/kpine/zwave-js-server-docker/issues/14#issuecomment-1221377377).

First upgrade: unknown version to 7.17.2
Second Upgrade: 7.17.2 --> 7.18.3

The version appears as 7.18.3 after the update everywhere including zwave-js-ui

**Comment by hvilander** [comment]
I was also able to update 7.17 to 7.18

I did have to remove a single quote out of the script line 124  
`print Activating the controllers bootloader. It will appear in about 10 seconds...`
The possessive on controller's, specifically.

I also used brew to install minicom & lrzsz.

I confirmed the version update on device info of the s2 stick in HA. 
This is on a Monterey 12.6.1 running on an x86 mac.
